src/cache.ts

```diff
@@ -59,27 +64,39 @@ export class LocalStorageCache implements ICache {
     );
 
     setTimeout(() => {
-      window.localStorage.removeItem(cacheKey);
+      const payload = this.getPayload(cacheKey);
+
+      if (!payload || !payload.body) return;
```

Comment:
is this the right style, or do we need to use braces?

Reply:
You're right, I've now applied a style consistant with the rest of the app.